content/docs/1.10.0/snapshots-and-backups/setup-a-snapshot.md

```diff
@@ -3,12 +3,76 @@
   weight: 1
 ---
 
-A [snapshot](../../concepts/#24-snapshots) is the state of a Kubernetes Volume at any given point in time.
+A [snapshot](../../concepts/#24-snapshots) is the state of a Kubernetes volume at any given moment in time.
 
-To create a snapshot of an existing cluster,
+## Snapshot Management via UI
 
-1. In the top navigation bar of the Longhorn UI, click **Volume.**
-2. Click the name of the volume of which you want a snapshot. This leads to the volume detail page.
+To create a snapshot of an existing cluster, follow these steps:
+
+1. In the top navigation bar of the Longhorn UI, click **Volume**.
+2. Click the name of the volume for which you want to create a snapshot. This leads to the volume detail page.
 3. Click the **Take Snapshot** button
 
-Once the snapshot is created you'll see it in the list of snapshots for the volume prior to the Volume Head.
\ No newline at end of file
+Once the snapshot is created, you will see it in the list of snapshots for the volume before the Volume Head.
+
+## Snapshot Management with Custom Resources (CRs)
+
+This section demonstrates how to create, list, restore, and delete Longhorn snapshots directly via `kubectl` using Longhorn’s **Custom Resources (CRs)**.
+
+> **Important Note**:  
+> Longhorn uses its own `Snapshot` CRD under the `longhorn.io` API group (for example, `v1beta2`), not the generic Kubernetes `VolumeSnapshot` from `snapshot.storage.k8s.io`.  
+
+### Create a Snapshot
+
+This section demonstrates how to create a Longhorn snapshot for an existing volume using `kubectl`.
+
+1.  **Prepare the manifest**: Create a file named `longhorn-snapshot.yaml`:
+
+    ```yaml
+    apiVersion: longhorn.io/v1beta2
+    kind: Snapshot
+    metadata:
+      name: longhorn-test-snapshot
+      namespace: longhorn-system
+    spec:
+      volume: pvc-d449abdc-5a17-4a80-a0ff-669173704060   # replace with your actual Longhorn volume name
+      createSnapshot: true
+    ```
+
+2.  **Apply the manifest**:
+
+    ```bash
+    kubectl apply -f longhorn-snapshot.yaml
+    ```
+
+    Expected output:
+
+    ```bash
+    snapshot.longhorn.io/longhorn-test-snapshot created
+    ```
+    
+    > **Note**: If the volume is detached, you will see a brief warning about the engine not running. Longhorn automatically retries, and the snapshot will complete once the volume is attached.
+
+### Listing Snapshots for a Volume
+
+List all snapshot CRs in the Longhorn namespace:
+
+```bash
+kubectl get snapshots.longhorn.io -l pvc-d449abdc-5a17-4a80-a0ff-669173704060 -n longhorn-system
+```
+
+### Deleting a Snapshot
+
+To remove the snapshot CR:
```

Comment:
Looks duplicated:
- `Deleting a Snapshot` already conveys the action.
- `To remove the snapshot CR` repeats the same idea.
Why not give a brief description, similar to `Create a Snapshot`?